tests/pages/remoteInstancesPage.js

```diff
@@ -224,12 +228,24 @@ module.exports = {
     I.seeElement(this.fields.userName);
   },
 
-  fillRemoteRDSMySQLFields() {
-    I.fillField(this.fields.userName, this.usernameRDSMySQL);
-    I.fillField(this.fields.password, this.passwordRDSMySQL);
-    I.fillField(this.fields.environment, 'RDS MySQL 5.6');
-    I.fillField(this.fields.cluster, 'rds56-cluster');
-    I.fillField(this.fields.replicationSet, 'rds56-replication');
+  fillRemoteRDSMySQLFields(serviceName) {
+    // eslint-disable-next-line default-case
+    switch (serviceName) {
+      case 'rds-mysql56':
+        I.fillField(this.fields.userName, this.usernameRDSMySQL);
+        I.fillField(this.fields.password, this.passwordRDSMySQL);
+        I.fillField(this.fields.environment, 'RDS MySQL 5.6');
+        I.fillField(this.fields.cluster, 'rds56-cluster');
+        I.fillField(this.fields.replicationSet, 'rds56-replication');
+        break;
+      case 'pmm-qa-postgres-12':
+        I.fillField(this.fields.userName, process.env.REMOTE_AWS_POSTGRES12_USER );
```

Comment:
code duplication can be avoided I believe

Reply:
I don't see the problem here. Do you have any suggestions for this? :)

Reply:
yeah, based on a service name you can have 2 objects something like `mysqlInputs` and `postgresqlInputs`. and then you will avoid code duplication

Reply:
You mean in array? :) I can try something.

Reply:
not array, but 2 separate objects with unputs

Reply:
I created two objects and a function to fill the fields. :)